app/backend-dotnet/Services/AzureOpenAITextCompletionService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Azure.AI.OpenAI;
+using Microsoft.SemanticKernel.AI;
+using Microsoft.SemanticKernel.AI.TextCompletion;
+
+namespace IMJohn.Services
+{
+    public class AzureOpenAITextCompletionService : ITextCompletion
+    {
+        private readonly OpenAIClient openAIClient;
+        private readonly string deployedModelName;
+
+        public AzureOpenAITextCompletionService(OpenAIClient openAIClient, string deployedModelName)
```

Comment:
I do not believe that this is DI-friendly, since it accepts a `string` as a ctor parameter. We might want to reconsider this.